### PR TITLE
"Happy Path" Update methods from the "e2e" project for simplifying and improve it's work

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Update methods from the "e2e" project for simplifying and improve their work

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/15445

### CODE REVIEW

File: tests/e2e/pageobjects/ide/DialogWindow.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+@injectable()
+export class DialogWindow {
+    private static readonly DIALOG_BODY_XPATH_LOCATOR: string = '//div[@id=\'theia-dialog-shell\']//div[@class=\'dialogBlock\']';
+    private static readonly CLOSE_BUTTON_XPATH_LOCATOR: string = `${DialogWindow.DIALOG_BODY_XPATH_LOCATOR}//button[text()='close']`;
+
+    constructor(@inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper) { }
+
+    async dialogDisplayes(): Promise<boolean> {
+        Logger.debug('WarningDialog.dialogDisplayes');
```
Comment (Katka92) [inline tests/e2e/pageobjects/ide/DialogWindow.ts]: Please replace all WarningDialog with DialogWindow in this class.

**Review by Katka92 — APPROVED** [review]

File: tests/e2e/pageobjects/ide/DialogWindow.ts (hunk 1 above)
Comment (Ohrimenko1988) [reply tests/e2e/pageobjects/ide/DialogWindow.ts]: Great catch, thank you.

**Comment by Ohrimenko1988** [comment]
ci-build

**Review by rhopp — CHANGES_REQUESTED** [review]
I guess the `package-lock.json` in root is mistake?
Other than that I'm +1.

**Review by rhopp — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
crw-ci-test

**Comment by Ohrimenko1988** [comment]
crw-ci-test

**Pull request merged**